tests/foreman/api/test_repositories.py

```diff
@@ -190,3 +193,28 @@ def test_positive_sync_kickstart_repo(self, module_entitlement_manifest_org, tar
     rh_repo = rh_repo.read()
     assert rh_repo.content_counts['package_group'] > 0
     assert rh_repo.content_counts['rpm'] > 0
+
+
+@pytest.mark.tier1
+def test_negative_upload_expired_manifest(module_org, target_sat):
+    """Upload an expired manifest and attempt to refresh it
+
+    :id: d6e652d8-5f46-4d15-9191-d842466d45d0
+
+    :Steps:
+        1. Upload a manifest
```

Comment:
```suggestion
        1. Upload an expired manifest
```

Reply:
we are not uploading an expired manifest here, we upload a manifest, then delete the subscription allocation causing the manifest to be expired.